scripts/dakota_input.py

```diff
@@ -20,6 +20,21 @@ def load_template(in_template):
 
 def render_input(xml_template, variable_dict, output_xml):
     """
+    Applies variables to the input file template
```

Comment:
```suggestion
    Formats the input file template's placeholder variables.
```
I think this might be a bit clearer.